packages/mds-trip-processor/src/config.ts

```diff
@@ -0,0 +1,152 @@
+export default {
```

Comment:
Same comment as earlier, I think this should be pulling from `mds-config`

Reply:
100% ... let it slide for this PR